src/components/Header/Avatar.tsx

```diff
@@ -35,49 +43,57 @@ function getInitials(name?: string) {
     .toUpperCase();
 }
 
-function handleSignOut() {
-  keycloackSessionLogOut().then(() => signOut({ callbackUrl: "/" }));
-}
-
 function Avatar({ user }: AvatarProps) {
+  const { width } = useWindowSize();
+  const screenSize = pixelWidthToScreenSize(width);
+
+  const username =
+    screenSize === "SM" || screenSize === "MD"
+      ? getInitials(user?.name)
+      : user?.name;
+
   return (
-    <DropdownMenu>
-      <DropdownMenuTrigger asChild>
-        <div className="flex h-8 w-8 cursor-pointer items-center justify-center rounded-full bg-info p-[8px] text-xs text-white shadow-sm transition-all duration-300 hover:bg-hover-color md:h-9 md:w-9 md:p-[10px] md:text-[13px]">
+    <div className="flex gap-x-2">
+      <div className="flex h-8 w-8 cursor-pointer items-center justify-center rounded-full bg-info p-[8px] text-xs text-white shadow-sm transition-colors duration-300 hover:bg-hover-color md:h-9 md:w-9 md:p-[10px] md:text-[13px]">
+        <Link href="/profile">
```

Comment:
Right now both the avatar and dropdown arrow are next to each other so its not fully clear that avatar in itself is a link and dropdown is a separate action of arrow.

Can you think of some other way to send user to profile with just one click